invisible_cities/reco/wfm_functions_test.py

```diff
@@ -57,6 +57,6 @@ def test_compare_cwf_blr():
 
     CWF = wfm.cwf_from_rwf(pmtrwf, range(NEVT), calib, deconv)
     diff = wfm.compare_cwf_blr(CWF, pmtblr,
-                               event_list=range(NEVT), window_size=200)
+                               event_list=range(NEVT), window_size=300)
 
-    assert max(diff) < 0.1
+    assert max(diff) < 0.15
```

Comment:
I talked about this with @jjgomezcadenas and he told me that `0.1` had been chosen with fine-tuning for that particular file. My understanding is that `0.15` being of the same order is fine, but tell me if you think otherwise.